.gitignore

```diff
@@ -4,3 +4,21 @@
 .vscode/*
 Cargo.lock
 .idea
+plotters/area_series.svg
+plotters/chart_builder_on.svg
+plotters/composable.svg
+plotters/configure_axes.svg
```

Comment:
Maybe change this list to a single entry `plotters/*.svg`  ?

Reply:
Oops, yes, thank you! 
Done.